setup.py

```diff
@@ -1,6 +1,9 @@
 import re
 
-from setuptools import setup
+try:
+    from setuptools import setup
```

Comment:
Why is this needed? 1 distutils iirc is deprecated and 2 setuptools comes with pip